src/Akeneo/Category/front/src/feature/components/channel/ChannelSelector.tsx

```diff
@@ -38,7 +23,7 @@ const ChannelSelector = ({value, values, onChange}: ChannelSelectorProps) => {
   return (
     <Dropdown>
       <SwitcherButton inline onClick={() => {open();}} label={translate('pim_common.channel')}>
-        <ChannelSpan code={`[${selectedChannel.code}]`} label={selectedChannel.labels[catalogLocale]} />
+        <span>{selectedChannel && getLabel(selectedChannel.labels, catalogLocale, `[${selectedChannel.code}]`)}</span>
```

Comment:
The function `getLabel` already handles wrapping the fallback between brackets

```suggestion
        <span>{selectedChannel && getLabel(selectedChannel.labels, catalogLocale, selectedChannel.code)}</span>
```

Reply:
Indeed, thx